src/Controllers/ImageController.php

```diff
@@ -18,6 +18,11 @@ public function __invoke(Request $request, string $size, string $file, string $w
     {
         abort_unless(in_array($size, config('imageresizer.sizes')), 400, 'The requested size is not whitelisted.');
 
+        if (Str::startsWith($file, 'sku:')) {
+            abort_unless(config('imageresizer.allow_sku'), 400, 'Retrieving image by SKU is not enabled.');
+            $file = $this->productImageUrlFromSku(Str::replaceFirst('sku:', '', $file));
```

Comment:
It might be different in my testing setup, but regardless of whether or not the image has already been generated before, it'll still always hit this controller. Makes sense, given that it uses the `storage` path which is not a public facing path, so it has to indirectly send this data to the user.

Is this different in production scenarios?